src/app/components/Sidebar/Sidebar.module.scss

```
@@ -0,0 +1,40 @@
+.wrapper {
+  margin-left: 10px;
+  background: #0c0f27;
+  color: white;
```

Try to keep one color convention in your project where possible.
```suggestion
  color: #fff;
```

---

Done